gum/backend-dbghelp/gumsymbolutil-dbghelp.c

```diff
@@ -133,7 +133,7 @@ gum_find_functions_matching (const gchar * str)
   if (dbghelp == NULL)
     return matches;
 
-  match_formatted_str = g_strdup_printf ("*!%s", str);
+  match_formatted_str = g_strdup_printf ("%s", str);
```

Comment:
Is this a breaking change? E.g. for scripts doing `DebugSymbol.findFunctionsMatching('CreateFileW')`.

Reply:
You are correct. Potentially this can break existing scripts. With original, "*!%s" string, I found that there is no way to specify  module for APIs. On Windows, same symol/function names can appear on multiple modules, the original implementation is more of random where the hook will be installed. For example, if you try to hook "CreateFileA", actually there are at least two exported functions from kernel32.dll and kernelbase.dll.  So, I suggest we should enforce the scripts to specify the module. If you are still uncomfortable, we can make it call to original implementation when there is no "!" symbol found in the symbol string.

Reply:
Answer for the questions above:

* symsrv.dll: this is a DLL used for symbol server operation. With this included, we can make Frida to load symbols from MSDL repository. Please check out documentation from [Using SymSrv](https://docs.microsoft.com/en-us/windows/win32/debug/using-symsrv)

* dbghelp-64.dll: I was testing 32 bit dll. I'll include latest DLLs for 64bit too.

Reply:
Makes sense. I think we should retain support for the old syntax so we don't break API. But we should make it clear in our docs that "loose" matching is deprecated. (After we've made sure it's supported by all backends.)

Regarding `symsrv.dll` – I guess we will need two copies with 32/64 suffixes and patch our `dbghelp-{32,64}.dll` to depend on them?

Reply:
Created new [PR](https://github.com/frida/frida-core/pull/320) for the distribution of symsrv files. I'll add old syntax support tomorrow and will update. Thanks.